etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -354,9 +366,11 @@
     "urth_section": {}
    },
    "source": [
-    "## Radar Widget\n",
+    "### Radar Widget\n",
     "\n",
-    "Make the radar chart an ipywidget so that it updates when the selected factors change."
+    "We will call a scotch a \"factor\" for this radar visualization.\n",
```

Comment:
@parente When studying the demo, I thought "factors" were the taste categories like smokey, honey, winey, etc. so I had a couple console.log to print out the "factors", and they were actually scotch names. Can you confirm the logic?